src/services/sanity.js

```diff
@@ -928,6 +928,12 @@ export async function fetchParentByRailContentId(railcontentId) {
  */
 export async function fetchMethods(brand) {
     const query = `*[_type == 'learning-path' && brand == '${brand}'] {
+    parent_content_data,
+    "breadcrumbs_data": parent_content_data[] {
+        "id": id,
+        "title": *[railcontent_id == ^.id][0].title,
```

Comment:
I'm not a sanity expert, but can you just go ^.title to get the top level data?  This query does not seem like it would be very efficient unless it does some magic internally.  Does this slow down the query at all?

Reply:
Nvm, parent_content_data is not a reference so this is probably the best we can do at the moment.